packages/ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@emcasa/ui",
+  "version": "0.1.0",
+  "description": "EmCasa's UI library",
+  "license": "MIT",
+  "main": "index.js",
+  "author": "EmCasa <dev@emcasa.com>",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "styled-system": "3.1.6"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
+}
```

Comment:
If we still have @emcasa/ui, why we need @emcasa/theme?
In case we're need to split up both, I'd rename @emcasa/theme to @emcasa/ui-theme

Reply:
We don't, I renamed `@emcasa/theme` to `@emcasa/ui` but forgot to replace it in the dependent packages.

Reply:
👍